data/src/androidTest/java/edu/nextstep/camp/data/MemoryDaoTest.kt

```diff
@@ -34,13 +32,13 @@ class MemoryDaoTest {
 
     @Test
     @Throws(Exception::class)
-    suspend fun writeMemoryAndReadInList() = withContext(Dispatchers.IO) {
+    fun writeMemoryAndReadInList() {
         val expression = Expression(listOf(10, Operator.Plus, 20)).toString()
         val result = "30"
-        val memory = Memory(expression, result, 0)
+        val memory = Memory(expression, result, 1)
         memoryDao.insert(memory)
 
-        val memoryById = memoryDao.getMemoryById(0)
+        val memoryById = memoryDao.getAll()[0]
```

Comment:
```suggestion
        val memoryById = memoryDao.getAll().first()
```
first를 사용해주시면 코드가 좀 더 보기 좋아져요

Reply:
아 맞네요. 자주 쓰지 않다보니 계속 까먹고 안쓰게 되네요. 피드백 감사드려요~! ㅎㅎ